aten/src/ATen/native/RNN.cpp

```diff
@@ -1182,20 +1670,37 @@ Tensor rnn_relu_cell(
 // an int8 or float16 quantized weight. This is advantageous in small-batch-size
 // scenarios where runtime is dominated by memory fetches of the weight matrix.
 
-std::tuple<Tensor, Tensor, Tensor> quantized_lstm(
-      const Tensor& _input, TensorList hx,
-      TensorList _params, bool has_biases,
-      int64_t num_layers, double dropout_p, bool train, bool bidirectional,
-      bool batch_first, c10::optional<ScalarType> dtype, bool use_dynamic) {
+std::tuple<Tensor, Tensor, Tensor> quantized_lstm_input(
+    const Tensor& _input,
+    c10::List<at::Tensor> hx_,
+    c10::List<c10::intrusive_ptr<CellParamsBase>> _params_,
+    bool has_biases,
+    int64_t num_layers,
+    double dropout_p,
+    bool train,
+    bool bidirectional,
+    bool batch_first,
+    c10::optional<ScalarType> dtype,
+    bool use_dynamic) {
+  auto hx = hx_.vec();
+  std::vector<QRNNCellParamsWrapper> params;
+  params.reserve(_params_.size());
+  for (const auto& param : _params_) {
+    params.emplace_back(static_cast<c10::intrusive_ptr<CellParamsBase>>(param));
+  }
   TORCH_CHECK(hx.size() == 2, "lstm expects two hidden states");
   if (at::cudnn_is_acceptable(_input)) {
     Tensor output, hy, cy;
-    lstm_cudnn_stub(_input.device().type(), output, hy, cy, _input, hx, _params, has_biases,
-                    num_layers, dropout_p, train, bidirectional, batch_first);
+    // TODOJAMES
```

Comment:
seems like this is not done yet?

Reply:
I think this whole part should be deleted, afaik CUDNN never supported quantized LSTMs

Reply:
then delete it and add TORCH_INTERNAL_ACCEPT(false) or just kill the entire conditional